scripts/disk_check.py

```diff
@@ -79,6 +81,14 @@ def test_writable(dirs):
             return False
         else:
             log_debug("{} is Read-Write".format(d))
+            space = os.statvfs(d)
+            if space.f_bavail < FREE_SPACE_THRESHOLD:
+                log_err("{} is no free space".format(d))
+                event_pub()
```

Comment:
>event_pub [](http://example.com/codeflow?start=16&length=9)

event_pub()  will send a read_only event, but this is RW branch. Is it conflicting event?

Reply:
Fixed, this needs a schema change: https://github.com/sonic-net/sonic-buildimage/pull/21351